code/game/gamemodes/cult/runes.dm

```diff
@@ -1,1053 +1,1006 @@
+			user.forceMove(get_turf(src)) //NO ESCAPE :^)
+		if(user.key)
+			user.visible_message("<span class='warning'>[user] slowly relaxes, the glow in their eyes dimming.</span>", \
+								 "<span class='danger'>You are re-united with your physical form. [src] releases its hold over you.</span>")
+			user.color = initial(user.color)
+			user.Weaken(3)
+			rune_in_use = 0
+			affecting = null
 			return
-		if(!(cultist.buckled || \
-			cultist.handcuffed || \
-			istype(cultist.wear_mask, /obj/item/clothing/mask/muzzle) || \
-			(istype(cultist.loc, /obj/structure/closet)&&cultist.loc:welded) || \
-			(istype(cultist.loc, /obj/structure/closet/secure_closet)&&cultist.loc:locked) || \
-			(istype(cultist.loc, /obj/machinery/dna_scannernew)&&cultist.loc:locked) \
-		))
-			user << "<span class='danger'>The [cultist] is already free.</span>"
-			return
-		if(cultist.buckled)
-			cultist.buckled.unbuckle_mob()
-		if (cultist.handcuffed)
-			cultist.handcuffed.loc = cultist.loc
-			cultist.handcuffed = null
-			cultist.update_inv_handcuffed()
-		if (cultist.legcuffed)
-			cultist.legcuffed.loc = cultist.loc
-			cultist.legcuffed = null
-			cultist.update_inv_legcuffed()
-		if (istype(cultist.wear_mask, /obj/item/clothing/mask/muzzle))
-			cultist.unEquip(cultist.wear_mask)
-		if(istype(cultist.loc, /obj/structure/closet)&&cultist.loc:welded)
-			cultist.loc:welded = 0
-		if(istype(cultist.loc, /obj/structure/closet/secure_closet)&&cultist.loc:locked)
-			cultist.loc:locked = 0
-		if(istype(cultist.loc, /obj/machinery/dna_scannernew)&&cultist.loc:locked)
-			cultist.loc:locked = 0
-		for(var/mob/living/C in users)
-			user.take_overall_damage(15, 0)
-			C.say("Khari[pick("'","`")]d! Gual'te nikka!")
-		qdel(src)
-	return fizzle(user)
-
-
-///Summon Cultist: Requires 3 cultists, summons a cultist to the rune's location and does 25 brute to all users.
-
-
-/obj/effect/rune/proc/cultsummon()
-	var/mob/living/user = usr
-	var/list/mob/living/carbon/cultists = new
-	for(var/datum/mind/H in ticker.mode.cult)
-		if (istype(H.current,/mob/living/carbon))
-			cultists+=H.current
-	var/list/mob/living/users = new
-	for(var/mob/living/C in orange(1,src))
-		if(iscultist(C) && !C.stat)
-			users+=C
-	if(users.len>=3)
-		var/mob/living/carbon/cultist = input("Choose the one who you want to summon", "Followers of Geometer") as null|anything in (cultists - user)
-		if(!cultist)
-			return fizzle(user)
-		if(cultist == user) //just to be sure.
+		if(user.stat == UNCONSCIOUS)
+			if(prob(10))
+				var/mob/dead/observer/G = user.get_ghost()
+				if(G)
+					G << "<span class='warning'>You feel the link between you and your body weakening... you must hurry!</span>"
+		if(user.stat == DEAD)
+			user.color = initial(user.color)
+			rune_in_use = 0
+			affecting = null
+			var/mob/dead/observer/G = user.get_ghost()
+			if(G)
+				G << "<span class='warning'><b>You suddenly feel your physical form pass on. [src]'s exertion has killed you!</b></span>"
 			return

```

Comment:
Probably need a `rune_in_use = 0` after the while, in case the loop stops b/c user became null.